### DISCUSSION TITLE
KievRPSSecAuth Cookie 填写解决方案

### DISCUSSION THREAD

**Discussion by Harry-zklcdc** [Harry-zklcdc/go-proxy-bingai#2] [General]
1. 前端页面添加设置项，请求的时候自动加上，用户需要自行填写
2. 自由部署 Cloudflare Worker，用户使用自己的 Worker

前置讨论: https://github.com/adams549659584/go-proxy-bingai/issues/363

**Comment by iseemylove** [comment]
看起来是第一个更方便？毕竟_U也是前端填写。或者直接复制完整的cookie呢？
目前看起来是必须登录用户才能使用是吗？

**Reply by Harry-zklcdc** [reply]
目前已知 `KievRPSSecAuth` **仅在Cloudflare上**需要，其他的待反馈

我的情况：
* 个人海外服务器上，直接运行二进制文件，可用情况**完全正常**，可未登录直接使用
* Cloudflare Worker 填写 `KievRPSSecAuth` 后**大部分可用**，可未登录直接使用，但无法使用 `/turing/conversation/create` 接口

**Reply by Rorschach331** [reply]
但无法使用 /turing/conversation/create 接口，用modheader改下user-agent，弄成手机端就能创建了，或者nginx反代的时候配置下

**Reply by Harry-zklcdc** [reply]
我去试试

**Comment by Harry-zklcdc** [comment]
Commit：https://github.com/Harry-zklcdc/go-proxy-bingai/commit/cb45b9b66b52dcb2a5c24f52783b64176570c9b9

保留了后端配置，同时增加了前端配置项
